src/frontend/hooks/server/deviceInfo.ts

```diff
@@ -21,10 +24,33 @@ export const useEditDeviceInfo = () => {
   return useMutation({
     mutationKey: ['device'],
     mutationFn: async (name: string) => {
-      return mapeoApi.setDeviceInfo({name});
+      return mapeoApi.setDeviceInfo({
+        name,
+        deviceType: deviceType ? expoToCoreDeviceType(deviceType) : undefined,
+      });
     },
     onSuccess: () => {
       queryClient.invalidateQueries({queryKey: [DEVICE_INFO_KEY]});
     },
   });
 };
+
+function expoToCoreDeviceType(d: DeviceType): DeviceInfo['deviceType'] {
+  switch (d) {
+    case DeviceType.PHONE: {
+      return 'mobile';
+    }
+    case DeviceType.TABLET: {
+      return 'tablet';
+    }
+    case DeviceType.TV: {
+      return undefined;
```

Comment:
leaving this as `undefined` seemed to make more sense than `UNRECOGNIZED`, but not super sure